### PR TITLE
RavenDB-15027 Display number of entries for time series

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/models/database/timeSeries/timeSeriesModel.ts [hunk 1]
```diff
@@ -36,7 +36,13 @@ class timeSeriesModel {
         });
         
         this.timestamp.extend({
-            required: true
+            required: true,
+            validation: [
+                {
+                    validator: () => this.timestamp().utc().format() !== "Invalid date",
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/timeSeries/timeSeriesModel.ts]: please never relay on magic string values. What if moment add locale support or we enable that? Code won't work. 

Looks like we can use this:
image

File: src/Raven.Studio/typescript/viewmodels/database/timeSeries/editTimeSeriesEntry.ts [hunk 2]
```diff
@@ -48,7 +48,10 @@ class editTimeSeriesEntry extends dialogViewModelBase {
         this.dateFormattedAsUtc = ko.pureComputed(() => {
             if (model.timestamp()) {
                 const date = moment(model.timestamp());
-                return date.utc().format(editTimeSeriesEntry.utcTimeFormat) + "Z (UTC)";    
+                if (date.utc().format() === "Invalid date") {
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/timeSeries/editTimeSeriesEntry.ts]: same here

**Review by ml054 — CHANGES_REQUESTED** [review]

File: src/Raven.Studio/typescript/models/database/timeSeries/timeSeriesModel.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/timeSeries/timeSeriesModel.ts]: done

File: src/Raven.Studio/typescript/viewmodels/database/timeSeries/editTimeSeriesEntry.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/timeSeries/editTimeSeriesEntry.ts]: done

**Review by ml054 — APPROVED** [review]

**Review by ml054 — APPROVED** [review]

**Pull request merged**